pkg/integration/testdata/exclude.json

```diff
@@ -1,3 +1,14 @@
 [
-
+  "/v2/classic/collections/:owner",
+  "/v2/callisto/collections/:owner",
+  "/v2/poa/collections/:owner",
```

Comment:
maybe instead of excluding we add list of tests that needs to run? i'm ok either way

Reply:
I think to add the tests to exclude is more easy, because we don't need to map all routes, only exclude what doesn't make sense.